### PR TITLE
Add upper bounding for targets

### PR DESCRIPTION
**Pull request by djiang3099**
Need to add polygon check for 'acceptable contours', oliver todo

### CODE REVIEW

File: vision.py [hunk 1]
```diff
@@ -117,26 +119,64 @@ def find_loading_bay(self, frame: np.ndarray):
...
+                    )
+                else:
+                    power_port_contour = acceptable_cnts[0]
                 power_port_points = get_corners_from_contour(power_port_contour)
                 # x, y, w, h = cv2.boundingRect(power_port_contour)
+                for i in range(4):
+                    cv2.circle(
+                        self.display, tuple(power_port_points[i][0]), 3, (0, 0, 255)
+                    )
+                cv2.imshow("Display", self.display)
```
Comment (raulvera59) [inline vision.py]: Oops. This one isn't in a test, and it doesn't have a timeout. This will hang the robot.

**Review by raulvera59 — CHANGES_REQUESTED** [review]
Found another show and waitkey, but not in test code and without a timeout. Will hang the robot.

**Review by raulvera59 — APPROVED** [review]
LGTM

**Review by Injector-Spenral — APPROVED** [review]

File: magic_numbers.py [hunk 2]
```diff
@@ -85,5 +87,11 @@
     [[0.13840045, -0.3277049, -0.00142985, -0.00095689, 0.28607425]], dtype=np.float32
 )
 
-ANGLE_SMOOTHING_AMOUNT = 0.3 # how much of the returned angle should be the last returned one
-DIST_SMOOTHING_AMOUNT = 0.8
\ No newline at end of file
+ANGLE_SMOOTHING_AMOUNT = (
+    0.3  # how much of the returned angle should be the last returned one
+)
```
Comment (Injector-Spenral) [inline magic_numbers.py]: Why is that now a tuple?

**Review by Injector-Spenral — CHANGES_REQUESTED** [review]

**Review by Injector-Spenral — APPROVED** [review]

**Pull request merged**